TCAT/Utilities/Endpoints.swift

```diff
@@ -108,12 +107,12 @@ class Network {
             "destinationName"   :   end.name,
             "originName"        :   start.name
         ]
-        
+
         // Add unique identifier to request
-        if let uid = userDefaults.string(forKey: Constants.UserDefaults.uid) {
+        if let uid = sharedUserDefaults?.string(forKey: Constants.UserDefaults.uid) {
```

Comment:
I just noticed, why is this now optional? I saw the global `let userDefaults = UserDefaults.standard` but do we then locally define `sharedUserDefaults`?